main.nf

```diff
@@ -20,60 +30,97 @@ nextflow.enable.dsl = 2
 // TODO nf-core: Remove this line if you don't need a FASTA file
 //   This is an example of how to use getGenomeAttribute() to fetch parameters
 //   from igenomes.config using `--genome`
-params.fasta = WorkflowMain.getGenomeAttribute(params, 'fasta')
-params.fasta_fai = WorkflowMain.getGenomeAttribute(params, 'fasta_fai')
+params.fasta = getGenomeAttribute('fasta')
+// params.fasta_fai = getGenomeAttribute(params, 'fasta_fai')
```

Comment:
Shouldn't this be before the inclusion of the other subworkflows?

Reply:
and isn't the fasta_fai needed?

Reply:
Right. There is something fishy there. Now that I look at it again, I don't like the way these params are turned into global variables. It makes it difficult to see where they are used. However, I see that we do the same thing in Sarek, so it is probably just an NF thing. 

Both `params.fasta` and `params.fasta_fai` are used in [workflows/bamtofastq.nf ](https://github.com/nf-core/bamtofastq/blob/e96b0095b62fd07e3125a24468dc47ae5d304159/workflows/bamtofastq.nf#L12-L13).

However, it seems to me that the definition [here](https://github.com/nf-core/bamtofastq/blob/e96b0095b62fd07e3125a24468dc47ae5d304159/workflows/bamtofastq.nf#L13) is redundant, because it is overwritten [here](https://github.com/nf-core/bamtofastq/blob/e96b0095b62fd07e3125a24468dc47ae5d304159/workflows/bamtofastq.nf#L75).

I'll make a PR with these corresponding changes.